spec/embiggen/uri_spec.rb

```diff
@@ -251,7 +251,24 @@ module Embiggen
       it 'returns false if the link is not shortened but uses a similar ' \
          'domain' do
         uri = described_class.new('http://notbit.ly/1ciyUPh')
+        expect(uri).to_not be_shortened
+      end
+
+      it 'returns false if the link is not shortened but uses a similar ' \
+         'domain without protocol' do
+        uri = described_class.new('notbit.ly/1ciyUPh')
+        expect(uri).to_not be_shortened
+      end
+
+      it 'returns false if the link is not shortened but uses a same TLD ' \
```

Comment:
As TLD stands for "top-level domain" and refers to the `.com` or `.co`, would it be more accurate to change this to `returns false if the link is not shortened but uses the same domain with a different TLD`?